pkg/project/sync.go

```diff
@@ -471,9 +472,13 @@ func syncFile(client utils.HTTPClient, projectID string, projectPath string, pat
 		return uploadResponse
 	}
 
+	mode := uint(fileStat.Mode().Perm())
+	if runtime.GOOS == "windows" {
+		mode = 509
```

Comment:
It would be clearer to use octal notation here, that's normal for file permissions. Can you change `509` to `0775` or `0o775`?